### PR TITLE
Add name and category metadata to opsrc and csc

### PR DESCRIPTION
**Pull request by kevinrizza**
-Update specs to include displayname and publisher as optional fields
-Set default values when new fields aren't specified
-Replace default opsrc with 3: redhat, certified and community
-Update default crs to have fields set correctly
-Add opsrc-provider label to categorize each default opsrc

### CODE REVIEW

File: manifests/03_operatorsource.crd.yaml [hunk 1]
```diff
@@ -29,6 +29,14 @@ spec:
...
     description: App registry namespace
     JSONPath: .spec.registryNamespace
+  - name: DisplayName
+    type: string
+    description: Display (pretty) name to indicate the OperatorSource's name
+    JSONPath: .spec.displayName
+  - name: Publisher
+    type: string
+    description: Publisher for resulting to CatalogSource
+    JSONPath: .spec.csPublisher
```
[the remark this answers is not in the document]
Comment (kevinrizza) [reply manifests/03_operatorsource.crd.yaml]: Addressed in latest commit

**Comment by aravindhp** [comment]
@jeff-phillips-18 please take a look at labels in the CR when you get the chance.

**Comment by jeff-phillips-18** [comment]
:+1 labels LGTM

**Pull request merged**